cmake/Scripts/GenerateRenameHeader.cmake

```diff
@@ -0,0 +1,20 @@
+include(${LOCATIONS_FILE})
+
+string(REPLACE " " ";" RENAME_HEADER_LIST  "${RENAME_HEADERS}")
+
+set(sse2_params 2 4 sse2)
+
+foreach(rename_header ${RENAME_HEADER_LIST})
+  if(${rename_header} MATCHES "sse")
```

Comment:
Yep, I don't like this very much either but I wanted to get it working asap. I'm fixing this now.